src/OWML.Utils/Container.cs

```diff
@@ -1,30 +1,31 @@
-﻿using Microsoft.Practices.Unity;
+﻿using Autofac;
```

Comment:
there's a nuget package "unity" which would let us not have to rewrite all of this? or did we already try that

Reply:
hmmm? dependency injection is rewritten and works, it's done :)